### PR TITLE
Feature/iesp 183

### PR DESCRIPTION
**Pull request by acolella92**

### CODE REVIEW

**Review by nttdata-rtorsoli — DISMISSED** [review]

File: src/main/java/it/pagopa/interop/probing/eservice/operations/repository/query/builder/EserviceContentQueryBuilder.java [hunk 1]
```diff
@@ -61,7 +62,7 @@ public Page<EserviceContent> findAllEservicesReadyForPolling(Integer limit, Inte
 
     return new PageImpl<>(pollingActiveEserviceContent,
         PageRequest.of(offset, limit,
-            Sort.by(ProjectConstants.ESERVICES_RECORD_ID_FIELD).ascending()),
+            Sort.by(Eservice_.ESERVICE_RECORD_ID).ascending()),
```
Comment (galales) [inline src/main/java/it/pagopa/interop/probing/eservice/operations/repository/query/builder/EserviceContentQueryBuilder.java]: does the `Eservice_` model ends with an underscore on purpose?

Comment (acolella92) [reply src/main/java/it/pagopa/interop/probing/eservice/operations/repository/query/builder/EserviceContentQueryBuilder.java]: it is authomatically generated by the hibernate-jpamodelgen dependency

Comment (nttdata-rtorsoli) [reply src/main/java/it/pagopa/interop/probing/eservice/operations/repository/query/builder/EserviceContentQueryBuilder.java]: Maybe is it EserviceView_ ?

Comment (acolella92) [reply src/main/java/it/pagopa/interop/probing/eservice/operations/repository/query/builder/EserviceContentQueryBuilder.java]: @nttdata-rtorsoli you're right, typo from copy and paste. Fix pushed

File: src/main/java/it/pagopa/interop/probing/eservice/operations/util/EserviceStatus.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  @Override
+  public String toString() {
+    return String.valueOf(value);
+  }
+
+  @JsonCreator
+  public static EserviceStatus fromValue(String value) {
+    for (EserviceStatus b : EserviceStatus.values()) {
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/eservice/operations/util/EserviceStatus.java:30]: Try to write with lambda function, using filter, findFirst throwing exception if Option.empty is returning (.orElseThrow(....)))

**Review by nttdata-rtorsoli — APPROVED** [review]
See comments

**Review by galales — APPROVED** [review]

**Pull request merged**